lua_modules/liquidcrystal/NTest_lc_i2c4bit.lua

```diff
@@ -0,0 +1,116 @@
+-- Run LiquidCrystal through some basic tests
+--
+--[[   The simplest method of testing leaves everything in RAM and is just
+
+  TCLLIBPATH=./tests/expectnmcu expect ./tests/tap-driver.expect \
+    ./lua_tests/mispec.lua \
+    ./lua_modules/liquidcrystal/liquidcrystal.lua \
+    ./lua_modules/liquidcrystal/lc-i2c4bit.lua \
+    ./lua_modules/liquidcrystal/NTest_lc_i2c4bit.lua
+
+Better to put NTest and LiquidCrystal in LFS, and then just run
+
+  TCLLIBPATH=./tests/expectnmcu expect ./tests/tap-driver.expect \
+    ./lua_modules/liquidcrystal/NTest_lc_i2c4bit.lua
+
```

Comment:
Maybe rather have instructions for use without expect or even better just list the required Lua files.